packages/grid/x-data-grid/src/tests/cells.DataGrid.test.tsx

```diff
@@ -184,6 +185,42 @@ describe('<DataGrid /> - Cells', () => {
     }).toWarnDev(['MUI: The cell with id=1 and field=brand received focus.']);
   });
 
+  it('selected cell  should alwys been in the dom,', function test() {
```

Comment:
Typos & orthograph errors. Can you also start it so it reads naturally like the other labels? E.g. `it('should keep selected ...')`